src/components/appcontent.tsx

```diff
@@ -43,9 +41,13 @@ const AppContent: FC<Props> = ({ showSidePane, showNavbar, appContent }) => {
             setShowSidePaneGrid={setShowSidePaneGrid}
             openSearchField={openSearchField}
             setOpenSearchField={setOpenSearchField}
+            searchValue={searchValue}
+            setSearchValue={setSearchValue}
           />
         )}
-        <div className="flex flex-col sm:flex-row mt-[3.5rem] ml-[0] bg-[#1C1C20]">
+        <div
+          className={`flex flex-col sm:flex-row mt-[56px] bg-[#1C1C20] ml-0`}
```

Comment:
Why backtick ?
Normal doube quotes will be enough.

Reply:
Updated